### ISSUE TITLE
Few-shot 3D LiDAR Semantic Segmentation for Autonomous Driving. (arXiv:2302.08785v1 [cs.RO])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#24834]
Few-shot 3D LiDAR Semantic Segmentation for Autonomous Driving. (arXiv:2302.08785v1 [cs.RO])<br>
https://ift.tt/lLkoTDy<br>
<p>In autonomous driving, the novel objects and lack of annotations challenge the traditional 3D LiDAR semantic segmentation based on deep learning. Few-shot learning is a feasible way to solve these issues. However, currently few-shot semantic segmentation methods focus on camera data, and most of them only predict the novel classes without considering the base classes. This setting cannot be directly applied to autonomous driving due to safety concerns. Thus, we propose a few-shot 3D LiDAR semantic segmentation method that predicts both novel classes and base classes simultaneously. Our method tries to solve the background ambiguity problem in generalized few-shot semantic segmentation. We first review the original cross-entropy and knowledge distillation losses, then propose a new loss function that incorporates the background information to achieve 3D LiDAR few-shot semantic segmentation. Extensive experiments on SemanticKITTI demonstrate the effectiveness of our method.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
